sceptre/helpers.py

```diff
@@ -211,3 +211,30 @@ def _detect_cycles(node, encountered_nodes, available_nodes, path):
             )
             encountered_nodes[dependency] = "DONE"
     return encountered_nodes
+
+
+def _call_func_on_values(func, attr, cls):
+    """
+    Searches through dictionary or list for objects of type `cls` and calls the
+    supplied function `func`. Supports nested dictionaries and lists.
+    Does not detect objects used as keys in dictionaries.
+
+    :param attr: A complex data structure to search through.
```

Comment:
This description of `attr` seems to obscure what the parameter is. My suggestion is to replace "A complex data structure" with "A dict or list". Otherwise, I take "complex data structure" to mean something much more complicated.